ipatests/test_integration/test_acme.py

```diff
@@ -111,6 +152,10 @@ def test_enable_acme_service(self):
         else:
             raise exc
 
+        result = self.master.run_command(['ipa-acme-manage', 'status'])
+        assert result.returncode == 0
```

Comment:
This check can be removed., the return code for status command will be 0 anyways.

Reply:
done